### PR TITLE
Adding "Fingerprint sensor or homepage button" post

### PR DESCRIPTION
**Pull request by gmtuca**

### CODE REVIEW

File: content/blog/2020-07-21-fingerprint-sensor-or-home-button/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Fingerprint sensor or home button?
+author: Arthur Ceccotti
+type: Post
+date: 2020-07-21
+description: When the success of your payment is based on how many nanometers your finger pressed a button 
+hero: "./hero.png"
```
Comment (gmtuca) [inline content/blog/2020-07-21-fingerprint-sensor-or-home-button/index.md:7]: needs changing

File: content/author.yaml [hunk 2]
```diff
@@ -21,6 +21,12 @@
   bio: I am a User Experience Designer with a mixed background of industrial design, service design, product design and user experience; currently working as a UX designer at a finacial services company.
   avatar: avatars/rb-avatar.jpeg
   twitter: Becky_Bayliss
+- id: Arthur Ceccotti
+  from: Brighton, UK
+  website: https://www.linkedin.com/in/arthur-ceccotti-0042/
```
Comment (gmtuca) [inline content/author.yaml:26]: I know, Sam... I will replace it with tuca.ninja soon!

Comment (slarsendisney) [reply content/author.yaml:26]: <3

File: content/index.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: Fingerprint sensor or home button?
```
Comment (gmtuca) [inline content/index.md]: wait.. let me get rid of this

**Pull request merged**